modules/postgresql/src/test/java/org/testcontainers/junit/postgresql/SimplePostgreSQLTest.java

```diff
@@ -17,6 +22,8 @@
 
 public class SimplePostgreSQLTest extends AbstractContainerDatabaseTest {
 
+    private static final String SEPARATOR = ";\n\n";
```

Comment:
It's a complicated question which separator sequence to use. Because:
1) `";"` is too trivial and does not cover complicated cases like stored procedures
2) we can introduce custom comment like `--GO` (ms sqlserver-style approach), but in this case each statement should be separated this way
3) If you do a `pg_dump` of the schema that has stored proc/func, you can see:
```
CREATE TABLE t (id int);

CREATE FUNCTION public.hi_lo(a numeric, b numeric, c numeric, OUT hi numeric, OUT lo numeric) RETURNS record
    LANGUAGE plpgsql
    AS $$
BEGIN
  hi := GREATEST(a, b, c);
  lo := LEAST(a, b, c);
END; $$;
```
Notice: in fact it uses two different separators: semicolon for regular DDL statement like create table or `$$;` for stored procedures. In my example test script I decided to use `";\n\n"` (not the same as `";"`), not great not terrible.